blocks/api/test/query.js

```diff
@@ -32,4 +32,21 @@ describe( 'query', () => {
 			expect( wp.element.renderToString( match ) ).to.equal( html );
 		} );
 	} );
+
+	describe( 'node()', () => {
+		it( 'should return a matcher function', () => {
+			const matcher = query.node();
+
+			expect( matcher ).to.be.a( 'function' );
+		} );
+
+		it( 'should return HTML equivalent WPElement of matched element', () => {
+			// Assumption here is that we can cleanly convert back and forth
+			// between a string and WPElement representation
+			const html = '<blockquote><p>A delicious sundae dessert</p></blockquote>';
+			const match = parse( html, query.node() );
+
+			expect( wp.element.renderToString( match ) ).to.equal( `<body>${ html }</body>` );
```

Comment:
@aduth The extra `body` here seemed weird to me. Is this something automatically added by the `parse` function or something?

Also in the test above, the `children` helper is returning the whole HTML while I expected it to return only the inner `p`.